.github/dependabot.yaml

```diff
@@ -9,3 +9,15 @@ updates:
     directory: '/'
     schedule:
       interval: 'weekly'
+  - package-ecosystem: "docker"
```

Comment:
Could you explain what are these changes?

Reply:
The changes in this file are to enable dependabot upgrades for dockerfiles and github actions.

Reply:
:+1: .
My assumption is that the way it work in those cases is that for Dockerfiles it checks from URLs in the FROM field and for github actions it checks for the action names in the defined actions. Let me know if that's not accurate.

I would have liked these changes to be independent from this PR as they are totally unrelated.

Reply:
> My assumption is that the way it work in those cases is that for Dockerfiles it checks from URLs in the FROM field and for github actions it checks for the action names in the defined actions. Let me know if that's not accurate.

That's correct. There are few PRs already opened. 

> I would have liked these changes to be independent from this PR as they are totally unrelated.

Ideally yes, but I preferred an option of having two atomic commits which works equally well.